modules/sdk/src/main/scala/org/tessellation/sdk/app/TessellationIOApp.scala

```diff
@@ -158,26 +164,26 @@ abstract class TessellationIOApp[A <: CliMethod](
                             .asResource
 
                           sdk = new SDK[IO] {
-                            val random = _random
-                            val securityProvider = _securityProvider
-                            val kryoPool = _kryoPool
-                            val metrics = _metrics
-                            val supervisor = _supervisor
-
-                            val keyPair = _keyPair
-                            val seedlist = _seedlist
-                            val generation = _generation
-                            val trustRatings = _trustRatings
-
-                            val sdkResources = res
-                            val sdkP2PClient = p2pClient
-                            val sdkQueues = queues
-                            val sdkStorages = storages
-                            val sdkServices = services
-                            val sdkPrograms = programs
-                            val sdkValidators = validators
-
-                            def restartSignal = _restartSignal
+                            val random: Random[IO] = _random
+                            val securityProvider: SecurityProvider[IO] = _securityProvider
+                            val kryoPool: KryoSerializer[IO] = _kryoPool
+                            val metrics: Metrics[IO] = _metrics
```

Comment:
why? if so, can you make such changes in separate PR to separate it from the current change (seedlist)?

Reply:
IntelliJ pointed these out as warnings.

Reply:
Per our offline discussion, these changes have been reverted.